stdlib/effect.mli

```diff
@@ -19,10 +19,18 @@
 type _ t = ..
 (** The type of effects. *)
 
+exception Unhandled : 'a t -> exn
+(** [Unhandled e] is raised when effect [e] is performed and there is no
+    handler for it. *)
+
+exception Continuation_already_taken
+(** Exception raised when a continuation is continued or discontinued more
+    than once. *)
+
 external perform : 'a t -> 'a = "%perform"
 (** [perform e] performs an effect [e].
 
-    @raise Unhandled if there is no active handler. *)
+    @raise Unhandled e if there is no handler for [e]. *)
```

Comment:
```suggestion
    @raise Unhandled if there is no handler for [e]. *)
```
Unfortunately `@raise Unhandled e if...` will be parsed as "`@raised`", "`Unhandled`", and "`e if ...`". On the other hand, the `[e]` was already mentioned in the beginning of the documentation, so I suppose it's okay to omit the first `e` in this sentence?

Reply:
@Octachron I think you might have missed this.

Reply:
I agree that omitting the exception payload is more in line with the rest of the documentation. This is fixed now.